app/models/manageiq/providers/openstack/infra_manager/ems_cluster.rb

```diff
@@ -28,11 +28,11 @@ def service_group_names
   end
 
   def service_group_services_running
-    service_group_services.where(SystemService.running_systemd_services_condition)
+    SystemService.running_systemd
```

Comment:
Thanks for the PR! Just a note: we want use an association instead of SystemService model to find matching system services.